content/en/docs/ops/common-problems/security-issues/index.md

```diff
@@ -93,9 +93,13 @@ matching requests should flow through. If all requests continue to be denied, yo
 1. Make sure that your authorization policies with ALLOW action don't use any HTTP only fields for TCP traffic.
 Otherwise, Istio ignores the ALLOW policies as if they don't exist.
 
+1. A HTTP Response with the value `RBAC: Access Denied` indicates an authorization policy is in effect. You can determine the authorization policy in effect by running `istioctl x authz check POD-NAME`.
```

Comment:
```suggestion
1. An HTTP response with the value `RBAC: Access Denied` indicates an authorization policy is in effect. You can determine the authorization policy in effect by running `istioctl x authz check POD-NAME`.
```

Reply:
Probably also specify the namespace `istioctl x authz check POD-NAME.POD-NAMESPACE` and mention this is available in 1.7.1 and later.

Reply:
fixed